common/src/main/java/fi/hel/verkkokauppa/common/configuration/ExperienceUrls.java

```diff
@@ -9,4 +9,7 @@
 public class ExperienceUrls {
     @Value("${order.experience.url:http://order-experience-api:8080/v1/order/}")
     private String orderExperienceUrl;
+
+    @Value("${payment.experience.url:http://payment-experience-api:8080/v1/payment/}")
+    private String paymentExperienceUrl;
```

Comment:
You can delete this because we get those urls straight from environment variable